scripts/start.js

```diff
@@ -347,6 +357,10 @@ function getUserInfo () {
     defaultInput: DEFAULT_DOMAIN
   });
 
+  var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
I can confirm that to `GET api/users` one needs System > Configure System > Manipulate Permissions permission which is not provided by default 'UserRole' role. Without that permission `<users />` is returned.

I guess it's still worth implementing (with fallback to user prompt) since this is for dev mode only and dev mode is often used with `admin@internal` that is compatible with guessing user id. Nevertheless I don't find it necessary to be part of this pull. For this pull we can go just with a user prompt.